apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/checkCapitalizedFields.kt

```diff
@@ -58,7 +58,7 @@ private fun ValidationScope.checkCapitalizedFields(selections: List<GQLSelection
         }
       }
       is GQLInlineFragment -> checkCapitalizedFields(it.selectionSet.selections)
-      is GQLFragmentSpread -> checkCapitalizedFields(fragmentsByName[it.name]!!.selectionSet.selections)
+      is GQLFragmentSpread -> fragmentsByName[it.name]?.let { fragment -> checkCapitalizedFields(fragment.selectionSet.selections) }
```

Comment:
```suggestion
      // it might be that the fragment is defined in an upstream module. In that case, it is validated
      // already, no need to check it again
      is GQLFragmentSpread -> fragmentsByName[it.name]?.let { fragment -> checkCapitalizedFields(fragment.selectionSet.selections) }
```

Reply:
💯